src/wide-character.js

```diff
@@ -66,7 +66,7 @@ const wideLatinLetterData: Array<[string, string, string]> = [
 
 const wideNumeralData: Array<[string, string, string]> = [
     ["mathbf", "textbf", "Main-Bold"],                // 0-9 bold
-    ["", "", ""],                         // 0-9 double-struck. No KaTeX font.
+    ["mathds", "", "AMS-Regular"],              // 0-9 double-struck. No KaTeX font.
```

Comment:
Is the comment still accurate? Is AMS-Regular not the font?

Reply:
Good question! I believe the comment is no longer accurate. I have updated it to just say `0-9 double-struck`